contact.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Sprint Challenge - Contact</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" media="screen and (min-width: 0px) and (max-width: 800px)" href="css/small.css">
+  <link rel="stylesheet" media="screen and (min-width: 801px) and (max-width: 1500px)" href="css/index.css">
+
+</head>
+
+<body>
+    <script>
+      function handleSubmit() {
+        window.open('thankyou.html');
+      }
+    </script>
+    <div class="container">
+      <header>
+        <div class="nav-bar">
+          <nav>
+            <a href="index.html"><img src="img/lambda-black.png" alt="Company logo."></a>
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="products.html">Products</a>
+            <a href="blog.html">Blog</a>
+            <a href="contact.html">Contact</a>
+          </nav>
+        </div>
+      </header>
+      <main>
+        <div class="form-contact-page">
+          <h1>Get in touch with us</h1>
+            <form onsubmit="handleSubmit()">
+              First name:<br><br>
```

Comment:
You're going to have fun when we start doing logins in react! good job playing around with functions!